src/extension.ts

```diff
@@ -2007,3 +2009,57 @@ async function kubernetesLint(document: vscode.TextDocument): Promise<void> {
     const diagnostics = ([] as vscode.Diagnostic[]).concat(...linterResults);
     kubernetesDiagnostics.set(document.uri, diagnostics);
 }
+
+async function cronJobRunNow(target?: any): Promise<void> {
```

Comment:
@bnookala Not sure what you're getting at here... we could type it as `string | ResourceNode | undefined`, but I feel like for these commands that can take parameters we've generally typed as `any`, because we can't statically verify where the command might be called from, and (admittedly inconsistently) checked expectations within the handler body.

Or were you suggesting we should switch to typing command handler args as `unknown` instead of `any`?